jdaviz/configs/default/plugins/data_quality/data_quality.py

```diff
@@ -92,7 +128,7 @@ def load_default_flag_maps(self):
 
     @property
     def unique_flags(self):
-        selected_dq = self.dq_layer.selected_obj
+        selected_dq = self.get_dq_layers()
```

Comment:
hmmm why doesn't `selected_obj` do what you want here?

Reply:
`get_dq_layers` did what I wanted and `selected_obj` didn't partly because I misunderstood the traceback that I was getting from `selected_obj`. It's clear to me now that this was because `selected_obj` can return types I wasn't expecting. In the next commit, I've added a property `self.dq_layer_selected_flattened` which just reorganizes the up the `selected_obj` output into the shape and types I expect.